packages/js/src/settings/routes/crawl-optimization.js

```diff
@@ -589,6 +589,8 @@ const CrawlOptimization = () => {
 							description={ __( "Limit the length of internal site search queries to reduce the impact of spam attacks and confusing URLs. Please enter a number between 1 and 50.", "wordpress-seo" ) }
 							disabled={ ! searchCleanup }
 							isDummy={ ! isPremium }
+							min={ 1 }
```

Comment:
When adding min and max to the input field it won't allow it to save changes and get the notification that you set in lines 68-69. So better to remove it, that will also remove the default html warning.

Reply:
Oh, good call! I must've missed something when trying this myself then 😞

Reply:
Reverted it and adapted PR text!

Reply:
CR & ACC :white: